File: Microservices/IoT.House.Automation.Microservices.Arduino/IoT.House.Automation.Microservices.Arduino.Application/MessageBroker/Events/ArduinoAddedEvent.cs
using System;
using System.Collections.Generic;
using System.Text;
using IoT.House.Automation.Microservices.Arduino.Domain.Models;

namespace IoT.House.Automation.Microservices.Arduino.Application.MessageBroker.Events
{
    public class ArduinoAddedEvent : Event
    {
        public ArduinoInfo Arduino { get; }
        public override Type Type => this.GetType();

        public ArduinoAddedEvent(ArduinoInfo arduino)
        {
            Arduino = arduino;
        }
    }
}

File: Microservices/IoT.House.Automation.Microservices.Mobile/application/src/main/java/iot/house/automation/microservices/mobile/application/messaging/events/ArduinoAddedEvent.java
package iot.house.automation.microservices.mobile.application.messaging.events;

import iot.house.automation.microservices.mobile.domain.models.arduino.Arduino;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class ArduinoAddedEvent extends BaseEvent {
    private Arduino arduino;

    public ArduinoAddedEvent(Arduino arduino) {
        this.arduino = arduino;
    }

    @Override
    public String getType() {
        return this.getClass().getSimpleName();
    }
}
